### DISCUSSION TITLE
Setting path...

### DISCUSSION THREAD

**Discussion by iknowl97** [DirectoryLister/DirectoryLister#757] [Help & Support]
Hello , I can't find where can I set path for DirectoryLister  can anyone help ? 
**### Thnxx**

**Comment by PHLAK** [comment]
I assume you're talking about a path that is listed. This is not possible to change due to the way the app works.

**Reply by iknowl97** [reply]
So where I need to place my "directory" to list ? I mean where is param for DIR - directory ?

**Reply by PHLAK** [reply]
To install Directory Lister

  1. [Download Directory Lister](https://www.directorylister.com/#download)
  2. Extract the zip/tar archive
  3. Copy the extracted files/folders (`index.php`, the `app` folder, etc.) to the directory you would like to list

**Reply by iknowl97** [reply]
I did it like this and it worked on Windows [XAMPP] but i can't set it up correct on my Ubuntu server with FTP
![Screenshot 2021-11-16 221627](https://user-images.githubusercontent.com/6195309/142042493-69168dd2-ed14-4f82-9bd4-0d3d590b3b90.png)

**Reply by PHLAK** [reply]
You are likely missing the `fileinfo` PHP extension. See [this section of the Common Issues documentation](https://docs.directorylister.com/help-and-support/common-issues#call-to-undefined-function-mime_content_type) for some additional info.

**Reply by iknowl97** [reply]
> You are likely missing the `fileinfo` PHP extension. See [this section of the Common Issues documentation](https://docs.directorylister.com/help-and-support/common-issues#call-to-undefined-function-mime_content_type) for some additional info.

Thank you i've to install it but i still get same error about  

```php
{
 $readmes->filter(static function (SplFileInfo $file) {
            return (bool) preg_match('/text\/.+/', (string) mime_content_type($file->getPathname()));
        })->sort(static function (SplFileInfo $file1, SplFileInfo $file2) {
            return $file1->getExtension() <=> $file2->getExtension();
        });
}
```

```
sudo apt install php-dom
```

already installed it on my Ubuntu. 

This is list of php modules: 

```
**php -m
[PHP Modules]
bcmath
Core
ctype
curl
date
dom
filter
ftp
gd
gettext
hash
iconv
intl
json
libxml
mbstring
mysqli
mysqlnd
openssl
pcntl
pcre
PDO
pdo_mysql
pdo_sqlite
Phar
posix
Reflection
session
shmop
SimpleXML
soap
sockets
sodium
SPL
sqlite3
standard
sysvsem
tokenizer
xml
xmlreader
xmlwriter
zip
zlib

[Zend Modules]**
```

**Reply by PHLAK** [reply]
I don't see `fileinfo` in your list of extensions. This means it was either installed incorrectly or it's installed but hasn't been enabled. You can try enabling it with `phpenmod fileinfo`. If that doesn't work try Googling around for how to enable this. Also, make sure you restart your web server (Apache, Nginx, etc.) after installing and enabling any PHP extensions to ensure the change is picked up.
